### ISSUE TITLE
geography cleanup

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8824]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

I don't think the attached (remainders from legacy messes) comply with https://handbook.arctosdb.org/documentation/higher-geography.html. They probably need individual attention, but let me know if I can help with something.

@mkoo

[temp_geog_probably_should_not_exist.csv](https://github.com/user-attachments/files/21005719/temp_geog_probably_should_not_exist.csv)

**Comment by dustymc** [comment]
I can update, but after oauth is out - it'll make duplicates, I'll just do it in the UI and deal with problems as they pop up.

**Comment by sharpphyl** [comment]
I'd vote for https://en.wikipedia.org/wiki/Regions_of_Finland

It's going to take a lot of work to clean up the mishmash we have.  Who is using these higher geographies?

**Comment by dustymc** [comment]
Geog folks, HELP?!?

**Comment by genevieve-anderegg** [comment]
Looking at this again and reviewing comments. So what we need to do is clean up the Level 1 values for Finland and Madagascar? (@mkoo above said the issues with Islands and Island Groups should be removed from the csv) Is that correct?

**Comment by dustymc** [comment]
I can push buttons with direction, I'm mostly hoping someone knows something about Finland (or someone picks some arbitrary schema...) and Madagascar and whatever.

Yes, I think we're removing island/island group per https://handbook.arctosdb.org/documentation/higher-geography.html

**Comment by genevieve-anderegg** [comment]
Reviewing Nepal, Finland, and Madagascar:

**Nepal**: The first level divisions are the [seven provinces of Nepal.](https://en.wikipedia.org/wiki/Provinces_of_Nepal#List_of_provinces_of_Nepal) GADM recognizes [five first level divisions](https://gadm.org/maps/FIN_1.html) of Nepal, based on the former [five Development regions of Nepal](https://en.wikipedia.org/wiki/Development_regions_of_Nepal), which were dissolved in 2015 with the formation of their new constitution. Therefore, we should reject GADM here and go with the current seven provinces of Nepal ([which our handbook allows for](https://handbook.arctosdb.org/documentation/higher-geography.html))

**Finland**: The first level divisions are  the [19 regions of Finland](https://en.wikipedia.org/wiki/Regions_of_Finland). GADM recognizes [five first level divisions](https://gadm.org/maps/FIN.html) that appear to be based on the six [Regional State Administrative Agencies](https://en.wikipedia.org/wiki/Regional_State_Administrative_Agency). However, the Wikipedia page itself states that "The agencies themselves do not constitute a subdivision of Finland ([ISO 3166-2:FI](https://en.wikipedia.org/wiki/ISO_3166-2:FI)); instead, the first-level subdivisions are the [regions of Finland](https://en.wikipedia.org/wiki/Regions_of_Finland)." Therefore, we should reject GADM here as well.

**Madagascar**: The Geography Committee reviewed Madagascar in May and [recommends the use of the current six provinces of Madagascar](https://github.com/ArctosDB/arctos/issues/8665#issuecomment-2852292381). GADM is up to date on [Madagascar's first level divisions.](https://gadm.org/maps/MDG_1.html)

@ArctosDB/geo-group does that sound good?

**Comment by dustymc** [comment]
@genevieve-anderegg thanks, I've updated documentation with these guidelines. Specifics below; I still have some questions.

# Language

Big-picture question: What title/language should we use, and does that scale to everything (probably with some specific exceptions, because there always are). I tentatively propose English (so "North Ostrobothnia" rather than "Pohjois-Pohjanmaa"), mostly for practical reasons: I don't think we can realistically deal with eg 愛媛県, and inconsistency is evil. If that (mostly) makes sense I'll add a "English version" clause to "Requested geography term must be present in GADM, IHO, or Wikipedia (when GADM or IHO are lacking)." in the documentation (which currently suggests "Åland" is correct). Help with that verbiage (if we're going that way) would be very appreciated.


# Nepal

This seems straightforward, everything is different. I will:

* add a 'previous geography' attribute to every locality that uses any Nepal-involved geography that's not just Nepal, then
* delete all Nepal-involved geography that's not just Nepal

Anyone who wants a finer-grained migration can request appropriate geography (template in the documentation) and move things around as they wish, before or after I do anything.

I'll plan to do this in the next couple weeks.


# Madagascar

Also seems straightforward. I will retain:

* https://arctos.database.museum/place.cfm?action=detail&geog_auth_rec_id=10000085
* https://arctos.database.museum/place.cfm?action=detail&geog_auth_rec_id=10000088

All other sub-Madagascar geography will be merged to Madagascar with a 'previous geography' attribute added in the next couple weeks.


# Finland

I think this is also direct - everything merges to "Finland" - because even the sorta-similar-named-things (https://en.wikipedia.org/wiki/Central_Finland vs https://arctos.database.museum/place.cfm?action=detail&geog_auth_rec_id=10000740) require more translation than I'm comfortable with, and in some cases (including that one) explicitly point to different **things** which just more-or-less share a name. That said, if anyone knows more than I do (maybe the various things with the same name are all the same shape???) and wants to propose any specific migrations I can accommodate that.

I'll plan to do this in the next couple weeks as well.

**Comment by dustymc** [comment]
done
